### PR TITLE
ED-2205 multi user account content

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: api/settings.py [hunk 1]
```diff
@@ -271,8 +271,14 @@
 FAB_FROM_EMAIL = os.getenv("FAB_FROM_EMAIL")
 FAB_OWNERSHIP_URL = os.getenv("FAB_OWNERSHIP_URL")
 FAB_COLLABORATOR_URL = os.getenv("FAB_COLLABORATOR_URL")
-OWNERSHIP_INVITE_SUBJECT = os.getenv("OWNERSHIP_INVITE_SUBJECT")
-COLLABORATOR_INVITE_SUBJECT = os.getenv("COLLABORATOR_INVITE_SUBJECT")
+OWNERSHIP_INVITE_SUBJECT = os.getenv(
+    'OWNERSHIP_INVITE_SUBJECT',
+    'Confirm ownership of {company_name}’s Find a buyer profile'
```
Comment (richtier) [inline api/settings.py]: I suspect we want `'` instead of `’`.

File: api/templates/account_collaborator_email.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
+{% load email_image from enrolment_email %}
```
Comment (richtier) [inline api/templates/account_collaborator_email.html]: can you add a print screen of this rendered template please.

File: api/templates/account_ownership_transfer_email.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+The Department for International Trade
+
+--------------------------------------
+
+https://www.linkedin.com/company/department-for-international-trade
+https://twitter.com/tradegovuk
+
+HM Government
+http://great.gov.uk
+Department for International Trade, 3 Whitehall Pl, Westminster, London SW1A
```
Comment (richtier) [inline api/templates/account_ownership_transfer_email.txt]: This template is very similar to the other .txt, and the HTML ones are mostly the same. Can you create some base templates and inherit from them, please.

File: api/settings.py (hunk 1 above)
Comment (sdonk) [reply api/settings.py]: I did copy and paste from the content I was given. I'll check with Dan

File: company/signals.py [hunk 4]
```diff
@@ -49,20 +49,20 @@ def save_case_study_change_to_elasticsearch(sender, instance, *args, **kwargs):
...
-        subject=instance.subject,
-        text_body=instance.invite_link,
-        html_body='<html><a href="{}">Click</a></html>'.format(
-            instance.invite_link
-        ),
-        recipient_email=instance.recipient_email,
-        from_email=FROM_EMAIL
-    )
+    notification = OwnershipChangeNotification(instance=instance)
+    notification.send()
```
Comment (richtier) [inline company/signals.py]: can you call this `send_async`, so the future developer will get a hint it happens asynchronously.

**Comment by sdonk** [comment]
screen shot 2017-09-22 at 16 47 48
screen shot 2017-09-22 at 16 47 23

**Pull request merged**